csv-validator-cmd/src/main/scala/uk/gov/nationalarchives/csv/validator/cmd/CsvValidatorCmdApp.scala

```diff
@@ -143,9 +146,9 @@ object CsvValidatorCmdApp extends App {
     case _ =>
   }
 
-  def getColumnFromCsv(csvFile: TextFile, csvSchemaFile: TextFile, columnName: String): List[String] = Try {
-    val validator = createValidator(true, Nil, false, false, false)
-    val csv = validator.loadCsvFile(csvFile, csvSchemaFile)
+  def getColumnFromCsv(csvFile: TextFile, csvSchemaFile: TextFile, columnName: String, maxCharsPerCell: Int): List[String] = Try {
```

Comment:
Do we have to send the maxCharsPerCell to the validator as well as when calling a method on the validator? 
If the validator knows about it, the method `loadCsvFile` can rely on using `this.maxCharsPerCell`

Reply:
Ah, yes! Good point. I think I was just trying to get it to work initially, and didn't go back to clean up.